docs/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ nav_order: 5
 
 ## main
 
+* Add documentation for JavaScript and CSS without webpacker
```

Comment:
```suggestion
* Add documentation for JavaScript and CSS without webpacker.
```

Reply:
Thanks for the suggestion Joel. It looks like Github [uses](https://viewcomponent.org/viewcomponents-at-github.html#integrating-javascript-behaviors) a [Catalyst](https://catalyst.rocks/guide/introduction/) approach instead of Stimulus. I have to admit, I wasn't aware of Catalyst at all and it's really interesting. 

It helps explain the shadow root web component approach described in [Encapsulating assets](https://viewcomponent.org/guide/javascript_and_css.html#encapsulating-assets). Separately, that section appears to be [unchanged](https://github.com/ViewComponent/view_component/blob/2b09e11076770830a91b1e356b639b948db3a931/docs/building_components.md#encapsulating-sidecar-assets) since the original gem documentation was released. When I came across it, my immediate tendency was to overlook it (and the Webpacker section) given the [Rails buzz](https://world.hey.com/dhh/modern-web-apps-without-javascript-bundling-or-transpiling-a20f2755) around import maps / Stimulus / eliminating node atm. Perhaps there's an opportunity to mention Catalyst or contextualise Web Components a bit more in that section to help justify it's approach?

I believe it was @reeganviljoen who [suggested](https://github.com/ViewComponent/view_component/issues/1064#issuecomment-2135904924) documentation for this would be useful. I was blocked for awhile trying to figure this out in lieu of anointed documentation. Now that Rails defaults have been updated to remove Webpacker and [include](https://github.com/rails/rails/blob/1b4c73e2a17e592b007e00f083ef4f9b0703ebcf/Gemfile#L19) stimulus-rails, I suspect I'm not the only one per the activity in Issue #1064.

ViewComponent's [Governance](https://viewcomponent.org/CONTRIBUTING.html#governance) documentation suggests there are several folks with commit access to this gem. Does GitHub have a process for allocating reviewers to pull requests? I'm not very familiar with the project prior to this so wouldn't know who to approach.